hack/update-license.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# don't add licenses to the site directory, it will break the docs
+# and will add them to the theme which is a submodule (bad)
+command -v addlicense || go install github.com/google/addlicense@v1.0.0
```

Comment:
Can you please reconcile it with https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/scripts/install-go-licenses.sh?